### ISSUE TITLE
Crash while loading the world

### ISSUE THREAD

**Issue by SeraphJACK** [judos/beltSorter#45]
Log: 
```
__beltSorter__/libs/migrations/util.lua:23: attempt to index field '?' (a nil value)
stack traceback:
    __beltSorter__/libs/migrations/util.lua:23: in function 'migrationAdd'
    __beltSorter__/migrations/beltSorter_0.6.2.lua:7: in main chunk
```

Maybe you should add

```lua
if force.recipes[recipeName] then
    ...
end
```

in [util.lua#L23](https://github.com/judos/beltSorter/blob/master/source/libs/migrations/util.lua#L23).

**Comment by judos** [comment]
wait that is still weird, if the given technology exists then also the recipes should be added for the beltSorter and therefore it should not crash :D can you provide the mods you use / savegame?

**Comment by SeraphJACK** [comment]
@yuanjie000

**Comment by yuanjie000** [comment]
[world.zip](https://github.com/judos/beltSorter/files/3364406/world.zip)

[mod list](https://paste.ubuntu.com/p/YKbBQs7b3m/)

**Comment by judos** [comment]
should be fixed with 0.6.4.
